x/lockup/client/cli/query.go

```diff
@@ -535,6 +536,47 @@ $ %s query lockup account-locked-longer-duration <address> <duration>
 	return cmd
 }
 
+// GetCmdAccountLockedDuration returns account locked records with a specific duration.
+func GetCmdAccountLockedDuration() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "account-locked-duration <address> <duration>",
+		Short: "Query account locked records with a specific duration",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Query account locked records with a specific duration.
+Example:
+$ %s query lockup account-locked-duration <address> <duration>
```

Comment:
I would remove this entirely  and utilize Cobra's `Example` field instead. We can then get rid of `Long` too.

Reply:
Done